org.gnumdk.syncpasswd.json

```diff
@@ -0,0 +1,127 @@
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libndp",
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://libndp.org/files/libndp-1.6.tar.gz",
+                    "sha256" : "0c7dfa84e013bd5e569ef2c6292a6f72cfaf14f4ff77a77425e52edc33ffac0e"
+                }
+            ]
+        },
+        {
+            "name" : "NetworkManager",
+            "buildsystem" : "meson",
+            "build-options" : {
+                "cflags" : "-ltinfo",
+                "cxxflags" : "-ltinfo"
+            },
+            "config-opts" : [
+                "-Dlibaudit=no",
+                "-Ddbus_conf_dir=/app/etc/dbus-1/system.d",
+                "-Ddnsmasq=/usr/bin/true",
+                "-Ddocs=false",
+                "-Dintrospection=true",
+                "-Diptables=/usr/bin/true",
+                "-Dlibpsl=false",
+                "-Dmodem_manager=false",
+                "-Dnmtui=false",
+                "-Dovs=false",
+                "-Dppp=false",
+                "-Dqt=false",
+                "-Dselinux=false",
+                "-Dsession_tracking=no",
+                "-Dsystemdsystemunitdir='no'",
+                "-Dsystemd_journal=false",
+                "-Dtests=no",
+                "-Dvapi=true"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "branch": "main",
+                    "url" : "https://gitlab.freedesktop.org/NetworkManager/NetworkManager.git"
+                }
+            ]
+        },
+        {
+            "name" : "mobile-broadband-provider-info",
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "branch": "main",
+                    "url" : "https://gitlab.gnome.org/GNOME/mobile-broadband-provider-info.git"
+                }
+            ]
+        },
+        {
+            "name" : "libnma",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dlibnma_gtk4=true",
+                "-Dgtk_doc=false",
+                "-Dintrospection=true",
+                "-Dvapi=true"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libnma.git",
+                    "branch" : "main"
+                }
+            ]
+        },
+        "python3-pyFxa.json",
+        "python3-requests-hawk.json",
+        "python3-pycryptodome.json",
+        {
+            "name" : "syncpasswd",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/gnumdk/syncpasswd.git",
+                    "branch": "main",
```

Comment:
```suggestion
```
this will cause the build to fail as soon as there is a new commit.

Reply:
this wasn't resolved

Reply:
Yes sorry, I need to plug my brain, will sync with how it's done in Lollypop: "tag" usage.